README.md

```diff
@@ -98,6 +98,20 @@ origin  https://github.com/YOUR_GITHUB_USER_NAME/stem-diverse-tv (push)
 upstream        https://github.com/anitab-org/stem-diverse-tv (fetch)
 upstream        https://github.com/anitab-org/stem-diverse-tv (push)
 ```
+## YouTube Data API Enabling
+
+**Steps**
+
+- You need a [Google Account](https://www.google.com/accounts/NewAccount) to access the Google API Console, request an API key, and register your application.
+- Create a project in the [Google Developers Console](https://console.developers.google.com/) and [obtain authorization credentials]      (https://developers.google.com/youtube/registering_an_application) so your application can submit API requests.
```

Comment:
The text `obtain authorization credentials` doesn't seem to be hyperlinked, you can remove the space or an extra enter to hyperlink the text :)

Reply:
@keshakaneria  Thank you, I will do it.